keps/sig-storage/2268-non-graceful-shutdown/README.md

```diff
@@ -0,0 +1,403 @@
+---
+title: Non graceful shutdown
+authors:
+  - "@yastij"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-node
+approvers:
+  - "@saad-ali"
+  - "@thockin"
+reviewers:
+  - "@andrewsykim"
+  - "@xing-yang"
+editor: Yassine Tijani
+creation-date: 2019-06-26
+last-updated: 2020-01-16
+status: implementable
+---
```

Comment:
This can be removed now that there's a kep.yaml file.

Reply:
Removed.